tests/phpunit/tests/rest-api/rest-search-controller.php

```diff
@@ -424,7 +424,7 @@ public function test_prepare_item() {
 	 */
 	public function test_prepare_item_limit_fields() {
 		if ( ! method_exists( 'WP_REST_Controller', 'get_fields_for_response' ) ) {
-			$this->markTestSkipped( 'Limiting fields requires the WP_REST_Controller::get_fields_for_response() method.' );
+			$this->fail( 'Limiting fields requires the WP_REST_Controller::get_fields_for_response() method.' );
```

Comment:
I think this needs to be reverted.

The only reason I can imagine that this condition exists, is to make backports of this test easier if the `WP_REST_Controller` class in an earlier iteration didn't have the `get_fields_for_response()` method.

If that is not the reason for the condition, then I'd say the whole condition block should likely be removed as the tests should fail without the method anyway.